### ISSUE TITLE
Repository creates url as `git+https://github.com...`

### ISSUE THREAD

**Issue by jasonkarns** [npm/init-package-json#56]
Recently, npm-init has begun generating packages where the repo url scheme is `git+https://`. Is this intentional or a bug? Why the change? (npm 3.5.3)

**Comment by adius** [comment]
I'm asking myself the same question!

**Comment by MrsJohnson20** [comment]
> Recently, npm-init has begun generating packages where the repo url scheme is `git+https://`. Is this intentional or a bug? Why the change? (npm 3.5.3)

**Comment by frank-dspeed** [comment]
This can be closed. Has no additional value everything is on https://npmjs.org.

# Solution
it is fully intentional also i need to inform you all that you should upgrade your nodejs installation and npm installations from time to time

**Comment by frank-dspeed** [comment]
@jasonkarns that means i saw indicators that there is some one upgrading npm without reading the changelog so i suggested fast and simple and general to update from time to time

but i understand now that the question was a diffrent so i will answer it finally

parsing string http://github.com/* gives me only enough information to know that i can get a tar.gz from there 
when i put git+https in the string i know i can use the git client to pull that even with credentials configured via the existing git client. 

git+ssh:// tells me that i can use git and also the ssh config to get credentials .

## Long Story short
if you work only with public repos where you have no need to push changes back or you do not need to use credentials for the download using https://github.com is fine all other need to use git@ git+ssh:// git+https
